SPECS/php/php.spec

```diff
@@ -292,27 +277,22 @@ package and the php-cli package.
 
 %package devel
 Summary:        Files needed for building PHP extensions
-License:        PHP AND Zend AND BSD AND MIT AND ASL 1.0 AND NCSA
-
+License:        PHP AND Zend AND BSD AND MIT AND ASL 1.0 AND NCSA AND Boost
 # always needed to build extension
 Requires:       autoconf
 Requires:       automake
 Requires:       gcc
 Requires:       gcc-c++
 # see "php-config --libs"
 Requires:       krb5-devel%{?_isa}
-Requires:       libedit-devel%{?_isa}
 Requires:       libtool
 Requires:       libxml2-devel%{?_isa}
 Requires:       make
-Requires:       openssl-devel%{?_isa} >= 1.0.1
+Requires:       openssl-devel%{?_isa} >= 1.0.2
 Requires:       pcre2-devel%{?_isa}
 Requires:       php-cli%{?_isa} = %{version}-%{release}
 Requires:       zlib-devel%{?_isa}
-
-Obsoletes:      php-pecl-json-devel < %{version}
-Obsoletes:      php-pecl-jsonc-devel < %{version}
-
+Recommends:     php-nikic-php-parser4 >= 4.13.0
```

Comment:
Do we have this?  Just thinking we may not be able to recommend it.  I don't see a subpackage getting created for it either

Reply:
So, when I'm reading the source code, it seems to be related to a patch (php-8.1.0-parser.patch) and adds an if check to see if this library is installed. It seems like a major thing for PHP developers but I don't think it's explicitly necessary either. https://github.com/nikic/PHP-Parser The patch just adds a check for if this is installed locally and uses it instead of the default parser.